kw

```diff
@@ -7,6 +7,16 @@ KW_INSTALL_PREFIX=${KW_INSTALL_PREFIX:-"##KW_INSTALL_PREFIX_TOKEN##"}
 
 KWORKFLOW=${KWORKFLOW:-"kw"}
 
+##UNINSTALLED##
```

Comment:
I I think that "UNINSTALLED" is not a good name for this functionality. Fwiu, you proposed this change to simplify the development mode cycle by enabling developers to use `./kw` and avoid `setup -i`. In this sense, I think that a name like "DEV_MODE" (or similar) better describes the nature of this code.